runtime/oti/vm_api.h

```diff
@@ -2171,7 +2176,8 @@ hashClassTableFree(J9ClassLoader* classLoader);
 * @param *classLoader
 * @param *className
 * @param classNameLength
-* @param flags
+* @param flags flags control whether to return a special class such as
```

Comment:
Please dont add this, if you do we need to add for all the other flags which will be numerous. Leave this as it was.